crawl-ref/source/rltiles/dc-mon.txt

```diff
@@ -137,11 +138,14 @@ jumping_spider MONS_JUMPING_SPIDER
 wolf_spider MONS_WOLF_SPIDER
 redback MONS_REDBACK
 orb_spider MONS_ORB_SPIDER
+broodmother MONS_BROODMOTHER
+culicivora MONS_CULICIVORA
```

Comment:
Think most of these (excepting broodmother and maybe steelbarb) should have `%rim 0` set, since they already have black borders drawn on.